activesupport/test/deprecation_test.rb

```diff
@@ -119,7 +119,7 @@ def test_raise_behaviour
     callstack = caller_locations
 
     e = assert_raise ActiveSupport::DeprecationException do
-      ActiveSupport::Deprecation.behavior.first.call(message, callstack)
+      ActiveSupport::Deprecation.behavior.first.call(message, callstack, "horizon", "gem")
```

Comment:
Alludes to backwards incompatibility: custom installed behaviors can raise ArgumentError's now.